scripts/configure_release_please/.release-please-manifest.json

```diff
@@ -0,0 +1,3 @@
+{
+    "packages/google-cloud-ids": "1.5.1"
```

Comment:
Is this file only here for testing? If so, could we move it under test_resources/?

Reply:
Fixed in 5c77eba